openshift4/core-web/base/deployment.core-web.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: apps/v1
```

Comment:
We don't have to include these files in this PR and in this Repo. 

All the K8S manifests can live in our https://github.com/bcgov-c/tenant-gitops-4c2ba9

This makes it easy to maintain